ts/features/design-system/core/DSToastNotifications.tsx

```diff
@@ -1,18 +1,114 @@
-import * as React from "react";
 import { Text as NBButtonText, Toast as NBToast } from "native-base";
-
+import * as React from "react";
 import ButtonDefaultOpacity from "../../../components/ButtonDefaultOpacity";
-import { DesignSystemScreen } from "../components/DesignSystemScreen";
-import { showToast } from "../../../utils/showToast";
-import { H2 } from "../../../components/core/typography/H2";
 import { VSpacer } from "../../../components/core/spacer/Spacer";
+import { H2 } from "../../../components/core/typography/H2";
+import { H3 } from "../../../components/core/typography/H3";
+import {
+  IOToast,
+  ToastNotification
+} from "../../../components/ui/ToastNotification";
+import { showToast } from "../../../utils/showToast";
+import { DesignSystemScreen } from "../components/DesignSystemScreen";
 
 export const DSToastNotifications = () => (
   <DesignSystemScreen title={"Toast Notifications (NativeBase)"}>
+    <H3 color={"bluegrey"} weight={"SemiBold"} style={{ marginBottom: 16 }}>
+      Events
+    </H3>
+
+    <ButtonDefaultOpacity
+      bordered={true}
+      block={true}
+      onPress={() => {
+        IOToast.show("Hello!", {
+          icon: "checkTic"
+        });
+      }}
+    >
+      <NBButtonText>Neutral</NBButtonText>
```

Comment:
I think it'd be best to avoid using NB components where possible; what do you think?

Reply:
Removed in https://github.com/pagopa/io-app/pull/4791/commits/74bca2f1fcfa93f286a347900bd1ef821f75f918